endstate_correction/simulation/base.py

```diff
@@ -116,7 +108,7 @@ def start(self):
         lamb = self.protocol.lam["ml-lambda"]
         self.logger.info(f"{lamb=}")
         # define where to store samples
-        trajectory_file = f"{base}/{self.name}_samples_{n_samples}_steps_{n_steps_per_sample}_lamb_{lamb:.4f}_{self.env}.dcd"
+        trajectory_file = f"{base}/{self.name}_samples_{n_samples}_steps_{self.protocol.timestep}_lamb_{lamb:.4f}_{self.env}.dcd"
```

Comment:
Why do we want the timestep to be in the trajectory name? I think usually we just use 1fs?
By the way, given the `self.name` is already in `{base}`, would it be good for the trajectory to just have name of `f"{base}/lamb_{lamb:.4f}.dcd"` all of the other information like `n_samples` could be easily retrieved from the dcd file?

Reply:
I'd prefer to have the `n_integration_steps` in the file name so that we can differentiate between different protocol lengths. I agree about the double naming and will remove that!